contracts/ethereum/contracts/Operator.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: GPL
+
+// Copyright 2022 Fluidity Money. All rights reserved. Use of this
+// source code is governed by a GPL-style license that can be found in the
+// LICENSE.md file.
+
+pragma solidity 0.8.11;
```

Comment:
Can we bump this up potentially and keep it consistent with the previous versioning?